frontend/src/components/allocation/list.js

```diff
@@ -1,53 +1,76 @@
 import React, { PropTypes } from 'react';
+import { DropdownButton } from 'react-bootstrap';
 import NomadLink from '../link';
 import FormatTime from '../format/time';
+import shortUUID from '../../helpers/uuid';
 import { renderDesiredStatus, renderClientStatus } from '../../helpers/render/allocation';
 
 const allocationStatusColors = {
-    complete: 'success',
-    running: 'info',
+    complete: '',
+    running: '',
     lost: 'warning',
     failed: 'danger',
 };
 
+const getAllocationNumberFromName = (allocationName) => {
+    const match = /[\d+]/.exec(allocationName);
+    return match[0];
+};
+
 const AllocationList = ({ allocations, nodes }) =>
   <table className="table table-hover table-striped">
     <thead>
       <tr>
+        <th></th>
         <th>ID</th>
         <th>Job</th>
         <th>Task Group</th>
-        <th>Task</th>
+        <th>Client</th>
         <th>Client Status</th>
-        <th>Desired Status</th>
-        <th>Node</th>
-        <th>Evaluation</th>
-        <th>Time</th>
+        <th>When</th>
+        <th></th>
       </tr>
     </thead>
     <tbody>
       {allocations.map((allocation) => {
           const color = allocationStatusColors[allocation.ClientStatus];
           return (
             <tr className={ color } key={ allocation.ID }>
+              <td>{ renderClientStatus(allocation) }</td>
```

Comment:
This icon seems a little bit 'lost' on the page to me and is basically redundant when you can see the client status in the same row. For instance, with the current icon (run) my first thought was that I could configure something on it... Maybe colorize the client status?